docs/api/breaking-changes-ns.md

```diff
@@ -0,0 +1,61 @@
+# Breaking changes (NetworkService)
+
+This document describes changes to Electron APIs after migrating network code
+to NetworkService API.
+
+Currently we don't have an estimate of when will we enable NetworkService by
+default in Electron, but as Chromium is already removing non-NetworkService
+code, the switch may come after a few major version releases.
```

Comment:
```suggestion
code, we might switch before Electron 10.
```

☝️ I'm not sure if that's true, but it's probably best to give an estimate that's unrelated to what the _current_ version is.

Reply:
10 looks good to me.